sktime/classification/base.py

```diff
@@ -129,17 +146,17 @@ def predict(self, X) -> np.ndarray:
         y : 1D np.array of shape =  [n_instances] - predicted class labels
         """
         self.check_is_fitted()
-
-        coerce_to_numpy = self.get_tag("coerce-X-to-numpy")
-        coerce_to_pandas = self.get_tag("coerce-X-to-pandas")
-        allow_multivariate = self.get_tag("capability:multivariate")
-        X = check_X(
-            X,
-            coerce_to_numpy=coerce_to_numpy,
-            coerce_to_pandas=coerce_to_pandas,
-            enforce_univariate=not allow_multivariate,
-        )
-
+        X = _internal_convert(X)
+        X = _internal_convert(X)
+        self.metadata_ = _check_classifier_input(X)
+        missing = self.metadata_["has_nans"]
+        multivariate = not self.metadata_["is_univariate"]
+        # This is incorrect needs to change
```

Comment:
why is this incorrect?

Reply:
its not, left the comment from before, thanks